api/v1/forkchoice.go

```diff
@@ -147,7 +147,7 @@ func (d *ForkChoiceNodeValidity) UnmarshalJSON(input []byte) error {
 
 // String returns a string representation of the ForkChoiceNodeValidity.
 func (d ForkChoiceNodeValidity) String() string {
-	if int(d) >= len(ForkChoiceNodeValidityStrings) {
+	if uint64(d) >= uint64(len(ForkChoiceNodeValidityStrings)) {
```

Comment:
What's the rationale for this change? I see this in a few other spots too.

Reply:
This change is coming from master (50179b8bd4d75c956248c8af698373fac1ddb6c9).
I've merged in master due to unrelated [linter issues](https://github.com/attestantio/go-eth2-client/actions/runs/10999190743/job/30538823000) that were already fixed in master.
sorry for the messed up diff.

Reply:
Ah okay, that makes sense. No worries!